src/js/player.js

```diff
@@ -2873,7 +2860,12 @@ class Player extends Component {
       playPauseKey = keydownEvent => (keycode.isEventKey(keydownEvent, 'k') || keycode.isEventKey(keydownEvent, 'Space'))
     } = hotkeys;
 
-    if (fullscreenKey.call(this, event)) {
```

Comment:
this changes the context that `fullscreenKey` is called

Reply:
Oh, right, the arrow function is just the default. Will change that back.